source/_data/data.json

```diff
@@ -6,6 +6,7 @@
   "bodyClass": "pl",
   "title": "Faker.words(2, true)",
   "text": "Faker.words(3, true)",
+  "cardimage" : "Faker.imageURL(318, 180, 'abstract', true, 'Some Text', true)",
```

Comment:
Let's take a look at using the holder.js library for this. You can see an implementation for this in the `_img.twig` file that needs some cleanup.